cms-adapter/src/Rsbc.Dmf.CaseManagement.Service/Services/DocumentService.cs

```diff
@@ -119,6 +119,64 @@ public async override Task<GetDmerReply> GetDmer(CaseIdRequest request, ServerCa
                 result.Item.Status = dmerCase.DmerStatus ?? string.Empty;
                 result.Item.Provider = new Provider();
                 result.Item.Provider.Name = dmerCase.Login?.FullName ?? string.Empty;
+                result.Item.DocumentId = dmerCase.DocumentId.ToString();
+                result.ResultStatus = ResultStatus.Success;
+                
+            }
+            catch (Exception ex)
+            {
+                result.ResultStatus = ResultStatus.Fail;
+                result.ErrorDetail = ex.Message;
+            }
+
+            return result;
+        }
+
+    
+
+        /// <summary>
+        /// Update ClaimDmer
+        /// </summary>
+        /// <param name="request"></param>
+        /// <param name="context"></param>
+        /// <returns></returns>
+        public async override Task<UpdateClaimReply> UpdateClaimDmer(UpdateClaimRequest request, ServerCallContext context)
+        {
+            var result = new UpdateClaimReply();
+
+            try
+            {
+                var loginId = Guid.Parse(request.LoginId);
+                var documentId = Guid.Parse(request.DocumentId);
+                var document = _documentManager.UpdateClaimDmer(loginId, documentId);
+                var mappedDocuments = _mapper.Map<Document>(document);
+                result.ResultStatus = ResultStatus.Success;
+            }
+            catch (Exception ex)
+            {
+                result.ResultStatus = ResultStatus.Fail;
+                result.ErrorDetail = ex.Message;
+            }
+
+            return result;
+        }
+
+        /// <summary>
+        /// Update UnClaimDmer
+        /// </summary>
+        /// <param name="request"></param>
+        /// <param name="context"></param>
+        /// <returns></returns>
+        public async override Task<UpdateClaimReply> UpdateUnClaimDmer(UpdateClaimRequest request, ServerCallContext context)
+        {
+            var result = new UpdateClaimReply();
+
+            try
+            {
+                var loginId = Guid.Parse(request.LoginId);
+                var documentId = Guid.Parse(request.DocumentId);
+                var document = _documentManager.UpdateUnClaimDmer(loginId, documentId);
+                var mappedDocuments = _mapper.Map<IEnumerable<Document>>(document);
```

Comment:
UpdateClaimReply.Item should only be one Document. Rename mappedDocument and also you forgot to assign it to the result e.g. result.Item = mappedDocument